spec/lib/bolt_spec/options.rb

```diff
@@ -7,6 +7,7 @@ module Options
     # generating the JSON schemas and validating config.
     def assert_type_description(definitions)
       definitions.each do |option, definition|
+        $stdout.puts option
```

Comment:
For debugging?

Reply:
From trying to figure out why a related test was broken. Just forgot to pull it out of a PR that was already merged. :/